geode-core/src/main/java/org/apache/geode/internal/cache/ColocationHelper.java

```diff
@@ -250,6 +250,11 @@ private static boolean ignoreUnrecoveredQueue(PartitionedRegion region, String c
     }
 
     String senderId = ParallelGatewaySenderQueue.getSenderId(childName);
+
+    if (region.getParallelGatewaySenderIds().contains(senderId)) {
+      return true;
+    }
+
```

Comment:
What's the reasoning behind this change?. 
According to the next `if` statement that was already part of the code, we can return `true` if and only if the list of `ParallelGatewaySenderIds` or `AsyncEventQueueIds` do not contain `childName`:

```
if (!region.getAsyncEventQueueIds().contains(senderId) && !region.getParallelGatewaySenderIds().contains(senderId) && IGNORE_UNRECOVERED_QUEUE) {
  return true;
}
```

Reply:
In scenario, when multi parallel GW senders are created on same region, and restarted, GW senders are hanging. I modified this part, and I will add test for it.